frontend/src/views/AmendingLawMetadataEditorRahmen.vue

```diff
@@ -386,11 +381,16 @@ watch(hasSaved, (finished) => {
               <fieldset class="contents">
                 <legend class="ris-label2-bold col-span-2">Dokumenttyp</legend>
 
-                <label :for="documentTypeId">Dokumenttyp</label>
-                <RisDropdownInput
-                  :id="documentTypeId"
+                <!-- eslint-disable-next-line vuejs-accessibility/label-has-for -->
+                <label id="documentTypeId" class="ris-label2-regular"
```

Comment:
Todo: should not use a hardcoded ID, use `useElementId` instead. See [styleguide](https://digitalservicebund.atlassian.net/wiki/spaces/VER/pages/884310020/Frontend+conventions+and+patterns#Avoid-using-IDs.-Do-not-assume-them-to-be-human-readable-or-static.). Same for the other IDs in this file